README.md

```diff
@@ -15,14 +15,14 @@ Installation
 We recommend the [Anaconda Python distribution](https://www.continuum.io/downloads).
 To install ``mne_bids``, you first need to install its dependencies:
 
-    $ pip install pandas mne
+    $ pip install numpy scipy matplotlib pandas mne
```

Comment:
you don't need to install these if you used anaconda, no?

Reply:
it might be good to mention the dependencies however for those who don't have anaconda installed. (I personally don't like using anaconda...)

Reply:
yes, I agree with @monkeyman192. Personally, I always use miniconda and thus I also have to install these packages. it's good to mention them and if they are already installed, it doesn't hurt to enter this command

Reply:
The problem is if anaconda already installs these packages and you try to do a `pip` install on top of it, things go bad and the install is corrupted. So, we cannot mix them. Advanced users can of course do their `pip` install without `anaconda` if they like ...

Reply:
okay fair enough, I'll make it something like "next to numpy, scipy, and matplotlib that are shipped with the standard anaconda distribution, you need the following: ..."